_layouts/default.html

```diff
@@ -92,4 +95,4 @@
     <!-- instant.page -->
     <script src="//instant.page/1.0.0" type="module" integrity="sha384-6w2SekMzCkuMQ9sEbq0cLviD/yR2HfA/+ekmKiBnFlsoSvb/VmQFSi/umVShadQI"></script>
   </body>
-</html>
+</html>
```

Comment:
You might want to fix the end of page changes between different editors to use the Linux ones

Reply:
I tried to fix it, please let me know if it didn't work.